examples/safer_cluster/main.tf

```diff
@@ -38,6 +38,18 @@ provider "kubernetes" {
   cluster_ca_certificate = base64decode(module.gke.ca_certificate)
 }
 
+// A random valid k8s version is retrived
+// to specify as an explicit version.
+data "google_container_engine_versions" "current" {
+  project  = var.project_id
+  location = var.region
+}
+
+resource "random_shuffle" "version" {
```

Comment:
nit: You might want to add `version = ">= 2.2.1"` to the Random Provider block for `random_shuffle` https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/745d30bfd823d653d3e4d73607576115ebf098bf/examples/safer_cluster/versions.tf#L31

Reply:
Added a constraint for latest major v.